conf/dynamicConfig.yaml.template

```diff
@@ -16,12 +16,15 @@
 #
 -  level: SYSTEM
    config:
-     celeborn.worker.directMemoryRatioToPauseReceive: 0.75
```

Comment:
seems this config item is not dynamic, I wonder whether it works now

Reply:
<img width="891" alt="image" src="https://github.com/user-attachments/assets/41cd8afc-8f7f-4366-a575-ceea4bebce35">

Currently, there are only 4 dynamic configs for quota management.

Reply:
> seems this config item is not dynamic, I wonder whether it works now

yes，seems can be dynamic，but not supported yet.

Reply:
Raised ticket to support more dynamic configs: https://issues.apache.org/jira/browse/CELEBORN-1598